src/views/PositionPdfPlot/useFetchCache.ts

```diff
@@ -82,7 +82,10 @@ const useFetchCache = <QueryType, ReturnType>(fetchFunction: (query: QueryType)
             dispatch({type: 'clear'})
         }
     }, [fetchFunction])
-    const get = useMemo(() => ((query: QueryType) => {
+    // The `get` function depends on the state, so it updates every time a reducer operation
+    // fires. This is intended: updating the `get` is how we trigger consumers that a fetch
+    // operation has completed and there's new data available in the cache.
+    const get = useCallback((query: QueryType) => {
```

Comment:
Converted the memoized function to a `useCallback`, which is semantically the same but probably expresses what we want to do more clearly.